postgres/templates/deployment.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "postgres.fullname" . }}
+  labels:
+    app: {{ template "postgres.fullname" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: {{ template "postgres.fullname" . }}
+  template:
+    metadata:
+      labels:
+        name: {{ template "postgres.fullname" . }}
+    spec:
+      containers:
+        - name: {{ template "postgres.fullname" . }}
+          imagePullPolicy: "Always"
+          image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+          ports:
+          - name: postgres
+            containerPort: 5432
+          volumeMounts:
+            - name: storage
+              mountPath: /var/lib/postgresql/data
+          env:
+          {{- with .Values.jupyterhub_db }}
+          - name: VRO_DB_JUPYTERHUB_USER
+            value: {{ .user }}
+          - name: VRO_DB_JUPYTERHUB_DB
+            value: {{ .db }}
+          {{- end }}
+            
+      volumes:
+        - name: storage
+          persistentVolumeClaim:
+            claimName: {{ template "postgres.fullname" . }}-physpvc
+        {{- with .Values.jupyterhub_db }}
+        - name: VRO_DB_JUPYTERHUB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "postgres.fullname" . }}-postgres
```

Comment:
This name seems a little odd. I think it would expand into `postgres-postgres`. I would have expected:
```
name: {{ template "postgres.fullname" . }}-secret
```
instead.

Reply:
I'll fix that.

Reply:
And actually based on Hub it probably shouldn't have a suffix at all.